TicTacToe.cpp

```diff
@@ -128,22 +128,39 @@ int TicTacToe::checkVictoryVertical(int teamSide) // check all (9) vertical colu
 
 void TicTacToe::viewAll()
 {
-	// Prints out the field with relevant team integers
-	std::cout << "    " << field[2][0][2] << " --- " << field[2][1][2] << " --- " << field[2][2][2] << std::endl;
+	int hold[26];   //create array to hold field integers
+	int count = 0;  //create counter to store integers in the array
+	for (int i = 0; i < 3; i++) {
+	//loop through the fieldand store integers
+		for (int j = 0; j < 3; j++) {
+			for (int k = 0; k < 3; k++) {
+				if (field[i][j][k] == 1 || field[i][j][k] == -1) {
```

Comment:
field[i][j][k] might not exist, it will return an error.

Reply:
I think the issues are addressed now.

Reply:
Well, likely not. You'd need to use `field.size()` instead because no matter what, if you try to access an element that doesn't exist with the array resolution operator `[]`, bad still *can* happen.

Reply:
Thank you for the clarification. Hopefully, it is adjusted now, please let me know if not.